src/app/setup-wizard/pre-visit/review-pre-visit-setup/company-details-summary/company-details-summary.component.html

```diff
@@ -2,7 +2,7 @@ <h5 class="bold w-100 text-center">
     <fa-icon [icon]="faBuilding" class="me-2"></fa-icon> {{company.generalInformation.name}}
 </h5>
 
-<table class="table table-bordered table-hover">
+<table class="table table-bordered table-hover table-sm">
     <tbody>
         <tr>
             <td class="w-25">
```

Comment:
It looks the address is not added in. better to comment it out if not included in alpha release,

Reply:
fixed